### DISCUSSION TITLE
SNR

### DISCUSSION THREAD

**Discussion by wa2zkd** [szpajder/dumphfdl#12] [Ideas]
any thoughts on making SNR from Modem available?

**Comment by szpajder** [comment]
Just implemented in devel branch (https://github.com/szpajder/dumphfdl/tree/devel).

**Reply by wa2zkd** [reply]
Great!!  Thanks

**Reply by wa2zkd** [reply]
that method of determining SNR is fine until the value  becomes small. IIRC 110C serial tone looks within the modem to calculate the SNR. I wonder what the arinc method is for this waveform.
